go/oasis-node/cmd/common/common.go

```diff
@@ -200,3 +202,15 @@ func LoadEntity(dataDir string) (*entity.Entity, signature.Signer, error) {
 	entitySignerFactory := fileSigner.NewFactory(dataDir, signature.SignerEntity)
 	return entity.Load(dataDir, entitySignerFactory)
 }
+
+func LoadSigner(signer string) (signature.Signer, error) {
+	switch signer {
+	case "ledger":
+		return ledgerSigner.NewSigner()
+	case "file":
+		_, signer, err := LoadEntity(flags.FileSignerDir())
+		return signer, err
+	default:
+		return nil, fmt.Errorf("signature format %s is not supported", signer)
```

Comment:
This is not a signature format but a signer, so the error should say something like `"signer %s is not supported"`.